pyproject.toml

```diff
@@ -158,9 +158,7 @@ Twitter = "https://twitter.com/twistedmatrix"
 allow-ambiguous-features = true
 
 [tool.hatch.version]
-source = "code"
-path = "src/twisted/__init__.py"
-search-paths = ["src"]
+source = "incremental"
```

Comment:
Refer to https://github.com/twisted/incremental?tab=readme-ov-file#using-hatchling